docs/devGuide/bootcamp/contributeToDocs.md

```diff
@@ -55,7 +55,7 @@ Ensure that you can serve the MarkBind documentation site in development mode. Y
 
 In the following steps, we will be fixing a hypothetical typo. You do not need to commit the changes or raise a PR, but understand the context and follow the steps to get a feel for the process.
 
-**!!Let's suppose we want to change the use of "Pop-Up" in our [user guide](../../userGuide/components/popups.html) to "Popup" (i.e. remove the hyphen).!!**
+**!!Let's suppose we want to change the use of "Pop-Up" in our <a tags="environment--combined" href="/userGuide/components/popups.html">user guide</a><a tags="environment--dg" href="https://markbind.org/userGuide/components/popups.html">user guide</a> to "Popup" (i.e. remove the hyphen).!!**
```

Comment:
If we are switching to the new combined tag, do we still need the anchor with `tags="environment --dg"`, and if so, in what situations do we need them?

^ This comment applies for all occurrences of this!

Reply:
Hi @lhw-1! We are not exactly switching to the new combined tag, we are adding a new combined tag to differentiate when using serving the `site.json` which displays both the ug and dg together and serving `dg-site.json` and `ug-site.json` individually as the individual dg and ug do not have access to the intra-linked files between them

Reply:
> Hi @lhw-1! We are not exactly switching to the new combined tag, we are adding a new combined tag to differentiate when using serving the `site.json` which displays both the ug and dg together and serving `dg-site.json` and `ug-site.json` individually as the individual dg and ug do not have access to the intra-linked files between them

👍 so there are three scenarios:
- DG + UG - markbind-master via netlify
- DG - markbind.org via github pages
- UG - markbind.org via github pages

Reply:
> Hi @lhw-1! We are not exactly switching to the new combined tag, we are adding a new combined tag to differentiate when using serving the `site.json` which displays both the ug and dg together and serving `dg-site.json` and `ug-site.json` individually as the individual dg and ug do not have access to the intra-linked files between them

Yep, that makes sense!

I think that just as a side note, the duplication of the anchor tags seem unnecessary - a method of declaring an alternate syntax for tags that do not provide such a way could be something to look into. For instance, it would be nice to be able to declare `<a tags="environment--combined" href="/userGuide/components/popups.html">user guide</a><alt tags="environment--dg" href="https://markbind.org/userGuide/components/popups.html" />`.

Reply:
> Yep, that makes sense!
> 
> I think that just as a side note, the duplication of the anchor tags seem unnecessary - a method of declaring an alternate syntax for tags that do not provide such a way could be something to look into. For instance, it would be nice to be able to declare `<a tags="environment--combined" href="/userGuide/components/popups.html">user guide</a><alt tags="environment--dg" href="https://markbind.org/userGuide/components/popups.html" />`.

I think thats a great idea! Do you want to open an issue to request that feature?